src/Nowruz/modules/layout/components/headerNavBar/useHeaderNavBar.tsx

```diff
@@ -89,6 +89,10 @@ export const useHeaderNavBar = () => {
     }
   };
 
+  const navigateToSettings =() => {
+    navigate('/nowruz/setting')
```

Comment:
Kindly use 'settings' instead of 'setting' for route to be compatible to V2 routes (it affects SEO)